### ISSUE TITLE
`typings install` fails on latest commit

### ISSUE THREAD

**Issue by mjesuele** [michaelgrosner/tribeca#93]
Hey! Amazing work on this. Even without running it yet, I can see the level of work and attention put in.

I just noticed that the latest commit seems to break `typings install`:

```
typings ERR! message Attempted to compile "lodash" as an ambient module, but it looks like an external module.
```

I don't know a ton of TypeScript but it looks like this is important:
https://github.com/michaelgrosner/tribeca/blob/master/typings.json#L17

I played around for a little bit trying to see if I could fix it but I didn't manage to.

Let me know if I can help!

**Comment by JamesBewley** [comment]
I'm also keen to experiment with this but hitting the same error when trying to build the docker image.

**Comment by michaelgrosner** [comment]
Sorry about this one. I must have introduced it in my last PR to update all the modules in Tribeca. I'll work on a fix now...

**Comment by michaelgrosner** [comment]
Take a look at the PR I just merged in #95 and let me know if that fixes it.
